leapp/logger/__init__.py

```diff
@@ -52,39 +52,66 @@ def _remote_emit(self, log_data):
 
 
 def configure_logger(log_file=None):
-    global _logger
-    if not _logger:
+    """
+    Configure loggers as per the description below.
+
+    logger: root
+      level: DEBUG
+      handler: StreamHandler
+        level: based on the debug/verbosity options
+      handler: LeappAuditHandler
+        level: DEBUG
+      logger: urllib3
+        level: WARN
+      logger: leapp
+        level: NOTSET
+        handler: FileHandler
+          level: DEBUG
+
+    :return: The 'leapp' logger
+    """
+    global _leapp_logger
+    if not _leapp_logger:
+        _leapp_logger = logging.getLogger('leapp')
+
         log_format = '%(asctime)s.%(msecs)-3d %(levelname)-8s PID: %(process)d %(name)s: %(message)s'
         log_date_format = '%Y-%m-%d %H:%M:%S'
-        path = os.getenv('LEAPP_LOGGER_CONFIG', '/etc/leapp/logger.conf')
+        logging.Formatter.converter = time.gmtime
```

Comment:
Please don't do this. This is a global state of all loggers. What about the one from the cookbook? (https://docs.python.org/3/howto/logging-cookbook.html#formatting-times-using-utc-gmt-via-configuration)
this could be solved something similar to:
```diff
diff --git a/etc/leapp/logger.conf b/etc/leapp/logger.conf
index ee1cf91..66102c4 100644
--- a/etc/leapp/logger.conf
+++ b/etc/leapp/logger.conf
@@ -10,7 +10,7 @@ keys=leapp_audit,stream
 [formatter_leapp]
 format=%(asctime)s.%(msecs)-3d %(levelname)-8s PID: %(process)d %(name)s: %(message)s
 datefmt=%Y-%m-%d %H:%M:%S
-class=logging.Formatter
+class=leapp.logger.UTCFormatter
 
 [logger_urllib3]
 level=WARN
```